From e9e12fd26bdb169739dd692280d3941d89ba425f Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Wed, 5 Jun 2019 16:23:25 +0200
Subject: [PATCH 3/6] kABI padding: usbcore
Patch-mainline: Never, kABI padding
References: bsc#1138132

Monitoring is important and costs nothing in the normal case
so we don't want to miss out on new techniques.

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 include/linux/usb/hcd.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/usb/hcd.h b/include/linux/usb/hcd.h
index ad60737..2eff0ab 100644
--- a/include/linux/usb/hcd.h
+++ b/include/linux/usb/hcd.h
@@ -669,6 +669,8 @@ struct usb_mon_operations {
 	void (*urb_submit_error)(struct usb_bus *bus, struct urb *urb, int err);
 	void (*urb_complete)(struct usb_bus *bus, struct urb *urb, int status);
 	/* void (*urb_unlink)(struct usb_bus *bus, struct urb *urb); */
+
+	void *suse_kabi_padding;
 };
 
 extern const struct usb_mon_operations *mon_ops;
-- 
2.16.4

